src/ui/layout/qgslayoutdesignerbase.ui

```diff
@@ -1585,9 +1586,15 @@
     <string>&amp;Keyboard Shortcuts...</string>
    </property>
   </action>
+  <action name="mActionPageProperties">
+   <property name="text">
+    <string>Page Properties…</string>
+   </property>
+  </action>
  </widget>
  <resources>
   <include location="../../../images/images.qrc"/>
+  <include location="../../../images/images.qrc"/>
```

Comment:
Remove ?

Reply:
this was added automatically by Qt Creator, didn't want to touch it.